internal/node/middlewarex/httpHandleMiddleware.go

```diff
@@ -0,0 +1,26 @@
+package middlewarex
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/labstack/echo/v4"
+)
+
+// DecodePathParamsMiddleware decodes path parameters.
+func DecodePathParamsMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		// Decode path parameters
+		paramValues := c.ParamValues()
+		for i, value := range paramValues {
+			decodedValue, err := url.QueryUnescape(value)
```

Comment:
should use `url.PathUnescape()`, because `+` is possible in RSSHub path.
https://github.com/golang/go/blob/master/src/net/url/url.go#L194